bin/spack-install-helper

```diff
@@ -2,25 +2,134 @@
 
 set -e
 
-POSSIBLE_TARGETS="daint-gpu daint-mc hohgant-cpu"
+# cuda compute capability:
+# P100 = 6.0
+# A100 = 8.0
+# H100 = 9.0
+
+POSSIBLE_TARGETS="daint-gpu daint-mc alps-zen2 alps-a100 alps-mi200 alps-gh200 alps-mi300a hohgant-cpu hohgant-nvgpu hohgant-amdgpu"
 DEFAULT_VARIANTS="+openmp +mpi build_type=Release"
 CUDA_EXTERNAL_DEF=""
+ROCM_EXTERNAL_DEF=""
+ROCM_ARCH=""
+CUDA_ARCH=""
+
+# TODO Remove this and hohgant-* below and update POSSIBLE_TARGETS when hohgant is shut down
+# Target aliases: print warning
+COLOR_DEFAULT='\033[0m'
+COLOR_YELLOW='\033[0;33m'
+COLOR_YELLOW_BOLD='\033[1;33m'
+COLOR_RED='\033[0;31m'
+COLOR_RED_BOLD='\033[1;31m'
+
+TEMPLATE="${COLOR_YELLOW_BOLD}Warning: the target ${COLOR_RED_BOLD}OLD${COLOR_YELLOW_BOLD} is deprecated and will be removed soon, please use ${COLOR_RED_BOLD}NEW${COLOR_YELLOW_BOLD} instead!!!${COLOR_DEFAULT}"
+
+OLD=""
```

Comment:
`target_machine="$1"`
And then use instead of `$1` `$target_machine`. And if the old name is matched, then update `$target_machine=alps-zen2/alps-a100/alps-mi200`